doc/sphinx/users-guide/increasing-your-hitrate.rst

```diff
@@ -383,3 +383,71 @@ So if you *really* need to vary based on 'User-Agent' be sure to
 normalize the header or your hit rate will suffer badly. Use the
 above code as a template.
 
+Cache misses
+------------
+
+When Varnish does not find an object for a request in the cache, then
+by default it performs a fetch from the backend on the hypothesis that
+the response might be cached. This has two important consequences:
+
+* Concurrent backend requests for the same object are *coalesced* --
+  only one fetch is executed at a time, and the other pending fetches
+  wait for the result. This is to prevent your backend from being hit
+  by a "thundering herd" when the cached response has expired, or if
+  it was never cached in the first place.  If it turns out that the
+  response to the first fetch is cached, then that cache object
+  satisfies the other pending requests.
```

Comment:
Yeah, the :ref: is two paragraphs further down ("Misbehaving servers"), which is a pretty good writeup on grace. Hm, will think about where it should go.